.github/workflows/test.yml

```
@@ -201,6 +201,16 @@ jobs:
           - 5432/tcp
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
+      postgres_isolated:
+        image: postgres:10
+        env:
+          POSTGRES_DB: tests
+          POSTGRES_USER: prisma
+          POSTGRES_PASSWORD: prisma
+        ports:
+          - 5435
```

```suggestion
          - 5435:5432
```

I think you can try that